source/agora/test/Base.d

```diff
@@ -847,6 +847,9 @@ public struct TestConf
     /// Number of nodes to instantiate
     size_t nodes = 4;
 
+    /// The value to give the `ValidatorCycle` of nodes
+    uint validator_cycle = 1008;
```

Comment:
This constant is a protocol-level value. So I think it should be maintained in the source code.

Reply:
I'm not sure I understand the suggestion ?